static/app/views/organizationGroupDetails/header.tsx

```diff
@@ -136,10 +136,6 @@ class GroupHeader extends React.Component<Props, State> {
 
     let className = 'group-detail';
 
-    if (group.isBookmarked) {
-      className += ' isBookmarked';
```

Comment:
👍 doesn't look like this is used in our CSS anywhere.

Reply:
I couldn't find any hits either. The other classnames nearby are still used.